src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/event.py

```diff
@@ -124,6 +120,52 @@ def event(self, ifname, vhid, event):
             if refresh:
                 self.middleware.call_sync('failover.status_refresh')
 
+    @private
+    async def restart_service(self, service, timeout, ha=False):
```

Comment:
Where is `ha` parameter used?

Reply:
Good point, it's not changed anywhere by callers. I could remove that.